### PR TITLE
Added doc about our logging, removed debug log level

### PR DESCRIPTION
**Pull request by slintes**
Signed-off-by: Marc Sluiter <marc@slintes.net>

### CODE REVIEW

File: docs/logging.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Logging
+
+For logging use the `Log` struct provided in the `log` package (see `pkg/log`).
+That struct offers several functions for providing logs with different log levels, verbosity and content.
+
+## Log Levels and Verbosity
+
+The Log struct offers several functions for logging with a decent log level: `Debug(msg)`, `Info(msg)`, `Warning(msg)`, `Error(msg)`, `Critical(msg)`.
```
Comment (rmohr) [inline docs/logging.md]: `Debug` should be dropped. We have verbosity in combination with Info. glog does not have it either and I personally find it confusing when to use what or what the difference should be (e.g at which verbosity level does debug start?).

Comment (slintes) [reply docs/logging.md]: Completely agree that debug is confusing as it is now. Will remove it.

Comment (slintes) [reply docs/logging.md]: @rmohr should I also remove it from code right now, and replace existing calls to it with `V(4).Info(...)`?

Comment (rmohr) [reply docs/logging.md]: Would be great.

**Comment by rmohr** [comment]
ok to test

**Comment by slintes** [comment]
@rmohr are there known issues with the test environments? Locally all tests pass.

**Comment by slintes** [comment]
retest this please

**Comment by rmohr** [comment]
@slintes So the results regarding to your code change are fine. In two cases we have well known flakyness (iscsi demo target does not come up and the no-error error from libvirt (error code 0). I have seen the 504 return error before but only added it right now to https://github.com/kubevirt/kubevirt/issues/1014. The same for the failing network test (where we seem to log a byte-array).

**Comment by rmohr** [comment]
@slintes the no-error error code issue may be interesting to look at. There seems to be an issue somewhere in libvirt-go. In the c-binding an error with code `0` means that there is no error. In the go-world `nil` is the equivalent. libvirt-go therefore tries to always return nil, but it seems that somewhere  a `0` error slips through.

**Pull request merged**